WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
> In Christianity, an antinomian is one who takes the principle of salvation by faith and divine grace to the point of asserting that the saved are not bound to follow the Law of Moses. The distinction between antinomian and other Christian views on moral law is that antinomians believe that obedience to the law is motivated by an internal principle flowing from belief rather than from any external compulsion.